src/metabase/driver/util.clj

```diff
@@ -49,7 +49,6 @@
       ;; actually if we are going to `throw-exceptions` we'll rethrow the original but attempt to humanize the message
       ;; first
       (catch Throwable e
-        (log/error e (trs "Database connection error"))
```

Comment:
When this log line was originally added in #10695, the `catch Throwable` block below didn't log the entire exception so we must log the original message here or we lose it. However, now all the `catch Throwable` block that calls this function ([1](https://github.com/metabase/metabase/blob/f90692d3d5ea685c78f96a98771914837c185f9e/src/metabase/driver/util.clj#L56-L57), [2](https://github.com/metabase/metabase/blob/f90692d3d5ea685c78f96a98771914837c185f9e/src/metabase/api/database.clj#L447-L448)) log the entire exception, we don't need this any more.